app/models/issue.rb

```diff
@@ -49,4 +51,9 @@ def sub_id
   def self.find_from_project(project, sub_id)
     Issue.find(project.issue_ids[sub_id.to_i - 1])
   end
+
+	def set_sub_id
+		current_id = project.issues.count
+		update_attribute(:sub_id, current_id)
+	end
 end
```

Comment:
Hi, your rubocop is failing. We use two spaces for indentation. If you are going to use tabs please configure your editor so that it uses 2 spaces as tabs.